tests/test_pythonpackage.py

```diff
@@ -0,0 +1,133 @@
+
+import os
+import shutil
+import sys
+import subprocess
+import tempfile
+
+from pythonforandroid.pythonpackage import (
+    get_package_dependencies,
+    get_dep_names_of_package,
+    get_package_name,
+    _get_system_python_executable,
```

Comment:
Minor: we import `_get_system_python_executable` why is it `protected`?

Reply:
mostly because it's a function nobody should ever need to use and which isn't particularly awesome :grimacing:

Reply:
Yes keep it that way, I like it being protected, but when I saw it being imported I got confused. Since it's only imported in the unit test, it's actually fine. It was nitpik anyway